src/config_crtrmodel.c

```diff
@@ -855,7 +860,7 @@ TbBool parse_creaturemodel_attributes_blocks(long crtr_model,char *buf,long len,
       game.conf.crtr_conf.spectator_breed = crtr_model;
   }
   // Set creature start states based on the flags
-  if ((crconf->model_flags & CMF_IsSpecDigger) != 0)
+  if ((crconf->model_flags & (CMF_IsSpecDigger|CMF_IsDiggingCreature)) != 0)
```

Comment:
Do the evil_start_state and good_start_state  also need to be cleaned up here?

Reply:
how so? good in this context should technically be roaming but think it's fine as is